templates/partials/financial-summary.html

```diff
@@ -6,7 +6,7 @@ <h2 id="section-1-header" tabindex="0">Financial Summary</h2>
       </div>
 
       {% if primary_committee or authorized_committees %}
-      <div class="page-subsection">
+      <div class="page-subsection page-subsection--noline">
```

Comment:
So this line is between primary and other committee:

![screen shot 2015-05-04 at 2 33 58 pm](https://cloud.githubusercontent.com/assets/1701077/7462976/8fb73086-f26a-11e4-81cc-5f30ce743faf.png)

Is that what you meant? I wanted to check before I re-added it.